src/main/resources/templates/home.html

```diff
@@ -19,8 +19,8 @@
                     <li>And much more...</li>
                 </ul>
             </div>
-            <a href="login.html" class="button">Login</a>
-            <a href="signup.html" class="button">Sign Up</a>
+            <a href="login" class="button">Login</a>
```

Comment:
when using a controller with springboot, there is no need to add .html extension on your links.